flow/packet-headers/rsvp.yaml

```diff
@@ -0,0 +1,1222 @@
+            default: 0
+            length: 8
+            features: [count]
+          x-field-uid: 3
+        as_number:
+          description: >-
+            Autonomous System number to be set in the ERO sub-object that this LSP should traverse through.
+            This field is applicable only if the value of 'type' is set to 'as_number'.
+            Note that as per RFC3209, 4-byte AS encoding is not supported.
+          type: integer
+          format: uint32
+          default: 0
+          maximum: 65535
+          x-field-uid: 4
+    Flow.RSVPExplicitRoute.Length:
+      type: object
+      properties:
+        choice:
+          description: auto or configured value.
+          type: string
+          default: auto
+          x-field-uid: 1
+          x-enum:
+            auto:
+              x-field-uid: 1
+            value:
+              x-field-uid: 2
+        auto:
+          description: >-
+            The OTG implementation can provide a system generated value for this property.
+            If the OTG is unable to generate a value the default value must be used.
+          type: integer
+          format: uint32
+          maximum: 256
+          default: 8
+          x-field-uid: 2
+        value:
+          type: integer
+          format: uint32
+          maximum: 256
+          default: 8
+          x-field-uid: 3 
+    Flow.RSVP.PathObjectsClassLabelRequest:
+      description: >-
+          C-Type is specific to a class num.
+      type: object 
+      properties:
+        c_type: 
+          $ref: '#/components/schemas/Flow.RSVP.PathObjectsLabelRequestCType'
+          x-field-uid: 1 
+    Flow.RSVP.PathObjectsLabelRequestCType:
+      description: >-
+        Object for LABEL_REQUEST class.
+        Currently supported c-type is Without Label Range (1).
+      type: object
+      properties:
+        choice:
+          type: string
+          x-field-uid: 1
+          default: without_label_range
+          x-enum:
+            without_label_range:
+              x-field-uid: 1
+        without_label_range:
+          $ref: '#/components/schemas/Flow.RSVP.PathLabelRequestWithoutLabelRange'
+          x-field-uid: 2
+    Flow.RSVP.PathLabelRequestWithoutLabelRange:
+      description: >-
+        Class = LABEL_REQUEST, Without Label Range C-Type = 1
+      type: object
+      properties:
+        reserved:
+          x-field-pattern:
+            description: >-
+              This field is reserved.  
+              It MUST be set to zero on transmission and MUST be ignored on receipt.              
+            format: integer
+            default: 0
+            length: 16
+            features: [count]
+          x-field-uid: 1
+        l3pid:
+          x-field-pattern:
+            description: >-
+              An identifier of the layer 3 protocol using this path. 
+              Standard Ethertype values are used.             
+            format: integer
+            default: 0
```

Comment:
Need to check the default ( might be IPv4 type is a better default )

Reply:
Update default as IPv4 ether type.